backend/test/todo.controller.js

```diff
@@ -21,3 +21,34 @@ describe("Stub test for controller and server", () => {
           .equal("test");
       }));
 });
+
+describe("test cases for create controller method", () => {
+  it("should add a new todo when body has all properties", () => {
+    const todo = {
+      name: "French homework",
+      description: "Pages 23 and 24 exercises 1-9",
+      isDone: false,
+      dueDate: "20-04-2021",
+    };
+    return request(Server)
+      .post("/api/todo")
+      .send(todo)
+      .then((r) => {
+        expect(r.statusCode).to.equal(201);
+      });
+  });
+
+  it("should send bad request when attempt to add new todo with missing name property", () => {
```

Comment:
Also, add test cases for the other missing required properties.

Reply:
Done